samples/gltf_viewer.cpp

```diff
@@ -312,9 +312,10 @@ int main(int argc, char** argv) {
 
     auto loadResources = [&app] (utils::Path filename) {
         // Load external textures and buffers.
+        std::string gltfPath = filename.getAbsolutePath();
         ResourceConfiguration configuration;
         configuration.engine = app.engine;
-        configuration.gltfPath = filename.getAbsolutePath().c_str();
+        configuration.gltfPath = gltfPath.c_str();
```

Comment:
How is that fixing the issue? You're still using a string on the stack.

Reply:
The std::string object now lives until the end of the scope, previously it was held in an unnamed transient object.   (as per the ResourceLoader docs, the cstring get cloned in its constructor)

ASAN aborted before, it passes now ... although it still emits lots of warnings in glslang.  :-\

Reply:
SG. Yeah glslang is scary.